schema/schema.hh

```diff
@@ -573,6 +573,11 @@ struct schema_static_props {
     bool wait_for_sync_to_commitlog = false; // true if all writes using this schema have to be synced immediately by commitlog
     bool use_schema_commitlog = false;
     system_table_load_phase load_phase = system_table_load_phase::phase1;
+    void enable_schema_commitlog() {
```

Comment:
I feel like this should have a better name, since it implies more things than just the schema commitlog.

If it turns out that we only do this for group0 tables, and we do this for all group0 tables (which is how I think it should be, we don't need this to be extremely fine-tuned), we should just call this setup_for_group0().

Also, we can consider moving it to the schema builder rather than applying it in a separate path.

All of this can be considered for a follow-up.

Reply:
> If it turns out that we only do this for group0 tables, and we do this for all group0 tables (which is how I think it should be, we don't need this to be extremely fine-tuned), we should just call this setup_for_group0().

It's not group0-specific, now we are turning it on for e.g. `system.local` and `system.scylla_local`

> Also, we can consider moving it to the schema builder rather than applying it in a separate path.

This would probably require adding the bool flag to the 'tables' system table. Now all the `schema_builder` describes should be convertible to mutations. These static configurators [were introduced](https://github.com/scylladb/scylladb/commit/3ef201d67a7521fc82b42e2ba992a16cc337ca5d) to be able attach a piece of config to an arbitrary table, including the one without schema_builder (`CDC_TOPOLOGY_DESCRIPTION`)

Reply:
I agree, it's better to keep this away from the schema.